tests/ipv6-evasion/ipv6-dos-mld-chiron/README.md

```diff
@@ -0,0 +1,7 @@
+# Description
+
+Detect an attack that cancels an arbitrary MLD subscription by becoming the MLD querier and faking the multicast specific queries.
+
```

Comment:
I do not get this attack :-/

Reply:
You can check the patch for snort3 as well https://redmine.cs.uni-potsdam.de/projects/snort3-ipv6-plugin/files

Reply:
Thanks for this pointer.
But it does not help me with this attack...
I do not know which rule triggers